src/runtime/auto-prompt-manager-test.js

```diff
@@ -2162,6 +2193,32 @@ describes.realWin('AutoPromptManager', (env) => {
     await tick(2);
   }
 
+  function resolveArticleExpectationOnce(
```

Comment:
Could we call the method `mockGetArticleResponse`? I think that would more clearly communicate the method's outcome
```suggestion
  function mockGetArticleResponse(
```